src/component/version.ts

```diff
@@ -18,13 +18,13 @@ import { mxClient } from './mxgraph/initializer';
 
 // WARN: this constant is automatically updated at release time by the 'manage-version-in-files.mjs' script.
 // So, if you modify the name of this file or this constant, please update the script accordingly.
-const libVersion = '0.40.0-post';
+const libraryVersion = '0.40.0-post';
```

Comment:
**issue:** please read the comment. It explains that the script updating the value **MUST** be updated!